popup/popup.js

```diff
@@ -142,10 +149,12 @@ function initPopup() {
           const note = (FIREFOX < 59 ? t('unreachableAMOHintOldFF') : t('unreachableAMOHint')) +
                        (FIREFOX < 60 ? '' : '\n' + t('unreachableAMOHintNewFF'));
           const renderToken = s => s[0] === '<'
-            ? $create('strong', {
+            ? $create('a', {
               textContent: s.slice(1, -1),
               onclick: handleEvent.copyContent,
-              tabIndex: -1,
+              href: '#',
+              className: 'copy',
+              tabIndex: 0,
```

Comment:
I think we only need to add `tabIndex` without changing `<strong>` to `<a>`?

Reply:
Again, we literally use links for accessibility hacks throughout the entire UI. This is already merged. I guess it showed up in the diff because the branch I pushed was slightly behind.